### ISSUE TITLE
WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail - insufficient contrast

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#681]
**Describe the bug**
WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail

To Reproduce:
run: 
 -  `pa11y --debug --reporter json --standard WCAG2AA <url of component>`

example:
 -  `pa11y --debug --reporter json --standard WCAG2AA --ignore WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2 http://localhost:9002/iframe.html?id=corecheckbox--checkboxlabeldisabled`


**Components samples with that error**

| component                           | selector                                                           | message                                                                                                                                                                                                                   |     |
| ----------------------------------- | ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
| checkboxlabeldisabled               | "#root > div:nth-child(2) > div > div > label > span:nth-child(2)" | "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.71:1. Recommendation:  change text colour to #737373." |     |
| checkboxonchangedisabled            | "#root > div:nth-child(2) > div > div > label > span:nth-child(2)" | "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.71:1. Recommendation:  change text colour to #737373." |     |
| radiobuttonlabeldisabled            | "#root > div:nth-child(2) > div > div > label > span:nth-child(2)" | "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.71:1. Recommendation:  change text colour to #737373." |     |
| radiobuttononchangedisabled         | "#root > div:nth-child(2) > div > div > label > span:nth-child(2)" | "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.71:1. Recommendation: change text colour to #737373."  |     |
| snackbar5 (Snackbar with an action) | "#client-snackbar > div:nth-child(2) > a"                          | "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.96:1. Recommendation:  change text colour to #0a61c8." |     |
